docs_src/microservices/application/AdvancedTopics.md

```diff
@@ -430,9 +430,17 @@ Application Services using the MessageBus trigger can request a background publi
 
 ```go
 
-func runJob (pub interfaces.BackgroundPublisher, ctxSeed interfaces.AppFunctionContext, done chan struct{}){
+func runJob (service interfaces.ApplicationService, done chan struct{}){
```

Comment:
Please add `!!! edgey "EdgeX 2.0" note somewhere that summarizes the Background Publisher changes for 2.0